components/datepicker/modules/calendar.vue

```diff
@@ -20,16 +20,16 @@
       <button
         v-for="(day, indexDays) in week.days"
         :key="indexWeek + indexDays"
+        :ref="el => { if (el) setDayRef(el, day.currentMonth) }"
         class="d-datepicker__day"
         :class="{
           'd-datepicker__day--disabled': !day.currentMonth,
-          'd-datepicker__day--selected': highlightDay ? ((day.text === highlightDay) && day.currentMonth) : day.selected,
+          'd-datepicker__day--selected': selectedDay ? ((day.text === selectedDay) && day.currentMonth) : day.selected,
         }"
         type="button"
-        :aria-label="day.text"
+        :aria-label="`${selectDayLabel} ${day.text}`"
```

Comment:
This should read the whole date, including month and year, or at least the month and the day